spec/my-inject_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'my-inject'
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)